packages/server/dataloader/atlassianLoaders.ts

```diff
@@ -122,11 +120,12 @@ export const allJiraProjects = (
             ...project,
             id: JiraProjectId.join(project.cloudId, project.key),
             userId,
-            teamId
+            teamId,
+            service: 'jira' as const
           }))
         })
       )
-      return results.map((result) => (result.status === 'fulfilled' ? result.value : null))
+      return results.map((result) => (result.status === 'fulfilled' ? result.value : []))
```

Comment:
whenever a dataloader return an array for a single key I like to return an empty array. Feels more like a database that way (return 0 rows vs. return null)